vocode/streaming/agent/state_agent.py

```diff
@@ -125,9 +126,10 @@ async def generate_completion(
 
         if self.resume:
             if human_input:
-                self.resume = await self.resume(human_input)
+                await self.resume(human_input)
             else:
-                self.resume = await self.resume(None)
```

Comment:
null states are not part of this data model, it's probably causing a lot of bugs